aks-node-controller/parser/templates/containerd.toml.gtpl

```diff
@@ -2,6 +2,10 @@ version = 2
 oom_score = -999{{if getHasDataDir .KubeletConfig}}
 root = "{{.KubeletConfig.GetContainerDataDir}}"{{- end}}
 [plugins."io.containerd.grpc.v1.cri"]
+  {{- if and ( .GetGpuConfig ) ( .GpuConfig.GetRequiresCdi ) }}
```

Comment:
@abenn135 aks-node-controller isn't currently being used, for this change to immediately take effect you'd want to edit the containerd config template here: https://github.com/Azure/AgentBaker/blob/master/pkg/agent/baker.go#L1392 which AgentBaker then uses o template this variable: https://github.com/Azure/AgentBaker/blob/master/parts/linux/cloud-init/artifacts/cse_cmd.sh#L157